tools/ci/tc/tasks/test.yml

```diff
@@ -82,6 +82,11 @@ components:
   browser-firefox:
     depends-on:
       - download-firefox-${vars.channel}
+    download-artifacts:
+      - task: download-firefox-${vars.channel}
+        artifact: public/results/firefox-${vars.channel}.*
```

Comment:
I'd suggest using a different name here. Perhaps `artifact-glob`, or just `glob`? There's so many `artifact*` in `download_artifacts` that it's a bit confusing.

Alternatively, could we put a concrete filename here instead of a pattern? It'd simplify `download_artifacts`, too.

Reply:
A concrete filename ends up being stronger coupling that causes problems if e.g. the archive format for a download changes. So that's why I went with a glob pattern.

Reply:
I was hoping that we could populate this field in the download task after we download the browser, but it seems like plumbing would be messy?

Reply:
Yeah, I'd rather leave that as a future enhancement.